src/app/settings/options.component.ts

```diff
@@ -101,4 +110,22 @@ export class OptionsComponent implements OnInit {
         }
         this.vaultTimeoutAction = newValue;
     }
+
+    async themeChanged(themeUpdate: string) {
```

Comment:
I think that the theme should not update until the user hits the "Save" button. I think the idea of previewing it without actually saving the changes it is confusing. It's also inconsistent with other options, which generally do not take effect until the user clicks the Save button. @cscharf might disagree though.

Reply:
We could get it to save the setting whenever this particular option is updated like in the browser, what do you think of doing this? It's beneficial to preview the theme first but i agree its confusing as you can click away from the settings whilst previewing the theme and easily forget to save it

Reply:
I think just apply and save the theme when the user clicks the Save button. I know other clients don't work this way (e.g. browser) but I think it's more important to be consistent with the other options on this page of this client.